test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -184,13 +184,13 @@ public void PackCommand_NewSolution_OutputInDefaultPaths()
                     ProjectFileUtils.WriteXmlToFile(xml, stream);
                 }
 
-                _dotnetFixture.RestoreSolutionExpectSuccess(testDirectory, solutionName, testOutputHelper: _testOutputHelper);
+                _dotnetFixture.RunDotnetExpectSuccess(testDirectory, $"restore {solutionName}.slnx", testOutputHelper: _testOutputHelper);
 
-                var nupkgPath = Path.Combine(projectFolder, @"bin\Debug", $"{projectName}.1.0.0.nupkg");
-                var nuspecPath = Path.Combine(projectFolder, @"obj\Debug", $"{projectName}.1.0.0.nuspec");
+                var nupkgPath = Path.Combine(projectFolder, @"bin\Release", $"{projectName}.1.0.0.nupkg");
+                var nuspecPath = Path.Combine(projectFolder, @"obj\Release", $"{projectName}.1.0.0.nuspec");
 
                 // Act
-                _dotnetFixture.PackSolutionExpectSuccess(testDirectory, solutionName, nuspecOutputPath: null, testOutputHelper: _testOutputHelper);
+                _dotnetFixture.RunDotnetExpectSuccess(testDirectory, $"pack {solutionName}.slnx", testOutputHelper: _testOutputHelper);
```

Comment:
I believe this PR removes all references to the `DotnetIntegrationTestFixture.PackSolutionExpectSuccess` and `DotnetIntegrationTestFixture.RestoreSolutionExpectSuccess` methods. We could either change them to support slnx, or let the caller pass the solution name, or remove these methods to clean up dead code (if they are indeed not referenced anymore, which I believe is true).